Formula/pugixml.rb

```diff
@@ -22,9 +23,11 @@ class Pugixml < Formula
   depends_on "cmake" => :build
 
   def install
-    system "cmake", ".", "-DBUILD_SHARED_AND_STATIC_LIBS=ON",
-                         "-DBUILD_PKGCONFIG=ON", *std_cmake_args
-    system "make", "install"
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+                    "-DBUILD_SHARED_LIBS=ON",
+                    "-DPUGIXML_BUILD_SHARED_AND_STATIC_LIBS=ON"
```

Comment:
Noticed that `mame` wasn't linking to `pugixml`.
`pugixml` only had static library and previous options output:
```
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_PKGCONFIG
    BUILD_SHARED_AND_STATIC_LIBS
```

I think pkgconfig files are always installed. New options are https://github.com/zeux/pugixml/blob/v1.12.1/CMakeLists.txt#L32-L35